site/source/docs/building_from_source/building_emscripten_from_source_on_linux.rst

```diff
@@ -21,73 +21,61 @@ Installing required tools
 
 These instructions explain how to install **all** the :ref:`required tools <toolchain-what-you-need>`. You can :ref:`test whether some of these are already installed <toolchain-test-which-dependencies-are-installed>` on the platform and skip those steps.
 
-1. Update the *system package manager* to ensure the package lists are up to date: 
+1. Update the *system package manager* to ensure the package lists are up to date::
```

Comment:
Looks like this PR changed `date:` (newlines) `::` to `date::`. The `::` on its own line is a sphynx thing we need, though, and the single `:` is needed for the english text, unless I'm missing something?

(happens a few more times below; easy to see with `?w=1` in the github url)

Reply:
Oops, yes I did clean that up at the same time.

This syntax is a shortcut:

http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks

"As a convenience, the "::" is recognized at the end of any paragraph. If immediately preceded by whitespace, both colons will be removed from the output (this is the "partially minimized" form). When text immediately precedes the "::", one colon will be removed from the output, leaving only one colon visible (i.e., "::" will be replaced by ":"; this is the "fully minimized" form)."

Reply:
Hmm, does it indent and format the next paragraph in the same way?